ui/app/controllers/allocations/allocation/task/fs.js

```diff
@@ -0,0 +1,38 @@
+import Controller from '@ember/controller';
+import { computed } from '@ember/object';
+import { filterBy } from '@ember/object/computed';
+import { isEmpty } from '@ember/utils';
+
+export default Controller.extend({
+  directories: filterBy('directoryEntries', 'IsDir'),
+  files: filterBy('directoryEntries', 'IsDir', false),
```

Comment:
Can you add `path`, `task`, `directoryEntries`, and `isFile` properties set to `null` in this file? It makes it easier to tell what all is coming from the route beyond the expected `model`.

Reply:
👍🏻 done, thanks for the reminder